tests/setup_database.py

```
@@ -98,17 +94,21 @@ def addCreator(spec, parent=None):
     :param parent: The parent of current document
     :type parent: dict
     """
+    userModel = loadModel('user')
     if 'creator' in spec:
-        spec['creator'] = users[spec['creator'].lower()]
+        spec['creator'] = userModel.findOne({'login': spec['creator'].lower()}, force=True)
     elif parent is not None:
 
         if parent.get('_modelType') == 'user':
             # if the parent is a user, then use that as the creator
             # (users don't have creators)
             spec['creator'] = parent
-        else:
+        elif parent.get('creatorId'):
             # otherwise, use the creator of the parent
-            spec['creator'] = userIds[parent['creatorId']]
+            spec['creator'] = userModel.load(parent.get('creatorId'), force=True)
+    else:
+        # if all else fails, just use any user for the creator
+        spec['creator'] = userModel.findOne({}, force=True)
```

Yeah, I guess that's what I'm worried about. I don't think Girder should have to support items/folders/collections having no creator unless that's something it already does. If it doesn't then I think this should raise an exception if there are no users found.